networkx/algorithms/dag.py

```diff
@@ -477,6 +507,36 @@ def is_aperiodic(G):
     NetworkXError
         If `G` is not directed
 
+    Examples
```

Comment:
Just a nit, but the lines in this section seem to have too few chars/line. One easy way to ensure the linebreaking is handled consistently with the rest of the library is to remove all the manual line-breaking and just let `black` do it for you (either by running black or via `pre-commit`).

Reply:
I pushed up a re-formatted version of your examples with longer line breaks that also modified the wording a bit - please check that the wording makes sense to you!